ontotext-yasgui-web-component/src/services/yasgui/configuration/configurator.ts

```diff
@@ -0,0 +1,16 @@
+import {CustomYasguiConfiguration} from '../../../models/custom-yasgui-configuration';
+
+/**
+ * The configuration of the yasgui component is divided into many configurators for easy maintenance and extension of yasgui.
+ * Every configurator have to implement {@see Configurator} interface.
+ */
+export interface Configurator {
+
+  // @ts-ignore
+  /**
+   * Apply configuration to <code>yasguiConfig</code> from <code>clientYasguiConfig</code>
+   * @param yasguiConfig - configuration which is passing to yasgui constructor.
+   * @param customYasguiConfig - configuration passed by client.
+   */
+  config(yasguiConfig, customYasguiConfig: CustomYasguiConfiguration);
```

Comment:
Where is the type of the `yasguiConfig` parameter?

Reply:
done